server/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java

```diff
@@ -676,8 +677,9 @@ public void testGroupByUnknownResultColumn() throws Exception {
     @Test
     public void testGroupByUnknownGroupByColumn() throws Exception {
```

Comment:
See original version https://github.com/crate/crate/blob/8ad9ecf201af3ef796d1d064bd7728444e57d1c8/server/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java#L672

Reply:
The error that is raised:

```
Caused by: SQLActionException: 400 Bad Request 4000 SQLParseException: Couldn't create executionContexts from
NodeOperations: [1=>NodeOp{ phase{id=1/distributed merge, nodes=[lRtAi887SCSp47-zAA2w-Q], dist=BROADCAST}, downstreamNodes=[pL8El7-xRqeJ54t7h40GKw], downstreamPhase=2, downstreamInputId=0}, 0=>NodeOp{ phase{id=0/collect, nodes=[lRtAi887SCSp47-zAA2w-Q], dist=MODULO}, downstreamNodes=[lRtAi887SCSp47-zAA2w-Q], downstreamPhase=1, downstreamInputId=0}]
Leafs: []
target-sources: [2=>[1], 1=>[0]]
original-error: UnsupportedOperationException: Cannot GROUP BY type: undefined
```

Is not an error that the user should ever see, as it points to a illegal execution plan. 

Although this change here is okay to fix the flakyness, I think we need to follow up here and also fix the error handling internally.

Reply:
I created a ticket for that in the board.

Reply:
So i think the error message should point out that it is impossible to group by the non-existing column `details_ignored['lol']` . Currently we get a type error which is misleading.